src/services/HapticService.ts

```diff
@@ -5,17 +5,17 @@ const options = {
   ignoreAndroidSystemSettings: false
 };
 
-const HapticMethods = [
-  'selection',
-  'impactLight',
-  'impactMedium',
-  'impactHeavy',
-  'notificationSuccess',
-  'notificationWarning',
-  'notificationError'
-];
+export enum HapticType {
+  selection = 'selection',
+  impactLight = 'impactLight',
+  impactMedium = 'impactMedium',
+  impactHeavy = 'impactHeavy',
+  notificationSuccess = 'notificationSuccess',
+  notificationWarning = 'notificationWarning',
+  notificationError = 'notificationError'
+}
 
-function triggerHaptic(hapticMethod: string, componentName: string) {
+function triggerHaptic(hapticMethod: HapticType, componentName: string) {
```

Comment:
It's intentionally kept its name `method` where all other places has changed to `type`?

Reply:
It was intentional, but on second thought it is better to be aligned with our 'old' code in the private repo.